website/docs/r/api_management_api.html.markdown

```diff
@@ -36,11 +36,6 @@ resource "azurerm_api_management_api" "example" {
   display_name        = "Example API"
   path                = "example"
   protocols           = ["https"]
-
-  import {
-    content_format = "swagger-link-json"
-    content_value  = "http://conferenceapi.azurewebsites.net/?format=json"
-  }
```

Comment:
We would like this to remain in the example so that users have a reference for the kind of input they can provide here.

We could perhaps swap the link in `content_value` out for a link that actually resolves to something. Not perfectly ideal but this could work:
 `https://raw.githubusercontent.com/hashicorp/terraform-provider-azurerm/refs/heads/main/internal/services/apimanagement/testdata/api_management_api_schema_swagger.json`

Reply:
Done.